### PR TITLE
asset: updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by freaky4wrld**
Updated CONTRIBUTING.md as per the requirement in the issue https://github.com/hackforla/ops/issues/114

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,147 @@
...
+- Agree to the AWS Customer Agreement and Service Terms, complete the registration by clicking `"Create Account and Continue"`, verify your phone number via text or call, confirm your email address following the instructions in the confirmation email, and finally sign in to access your new AWS account using your email and password.
 
-- Explain your process.
\ No newline at end of file
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+### **Installing Terraform**
+
+We are using the [Official HashiCorp install instructions](https://developer.hashicorp.com/terraform/install) for installing terraform.
```
Comment (chelseybeck) [inline CONTRIBUTING.md:40]: ``
```suggestion
Use the [Official HashiCorp install instructions](https://developer.hashicorp.com/terraform/install) for installing terraform.
```
There's nothing really wrong here - just removing pronouns to standardize and simplify

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,147 @@
-## Local Development/Installation instructions
+# **How to Contribute to DevOps**
```
Comment (chelseybeck) [inline CONTRIBUTING.md:1]:
```suggestion
# **How to Contribute to DevOps Security**
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,20 +1,147 @@
...
+### **Installing Terraform docs**
+
+Follow the Terraform docs [installation guide](https://terraform-docs.io/user-guide/installation/)
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Fork the repository**
+
+You can fork the hackforla/devops-security repository by clicking <a href="https://github.com/hackforla/devops-security/fork"> <button> <img src="https://user-images.githubusercontent.com/17777237/54873012-40fa5b00-4dd6-11e9-98e0-cc436426c720.png" width="8px"> Fork</button></a>
```
Comment (chelseybeck) [inline CONTRIBUTING.md:54]:
```suggestion
Fork the hackforla/devops-security repository by clicking <a href="https://github.com/hackforla/devops-security/fork"> <button> <img src="https://user-images.githubusercontent.com/17777237/54873012-40fa5b00-4dd6-11e9-98e0-cc436426c720.png" width="8px"> Fork</button></a>
```

**Review by chelseybeck — DISMISSED** [review]
This is a great start to our contributing guide, @freaky4wrld! Thanks so much for working on this 😄

**Comment by freaky4wrld** [comment]
@chelseybeck document is updated as required, give your feedback when you can thanks :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,20 +1,149 @@
...
-### Working with issues
+**The team recommends using [VS Code](https://code.visualstudio.com/download) as the preferred text editor for working on code, but feel free to utilize a text editor of your preference.**
 
-- Explain how to contribute to an existing issue.
+**If you have any additional questions about your contribution process, please feel free to reach out to the team in the [#ops](https://hackforla.slack.com/archives/CV7QGL66B) Slack channel.**
+<br><br>
 
-### Working with forks and branches
+## **Table of Contents**
+- [**How to Contribute to DevOps**](#how-to-contribute-to-devops)
```
Comment (chelseybeck) [inline CONTRIBUTING.md:14]:
```suggestion
- [**How to Contribute to DevOps Security**](#how-to-contribute-to-devops-security)
```

**Review by chelseybeck — APPROVED** [review]

**Review by chelseybeck — APPROVED** [review]

**Review by erniep278 — APPROVED** [review]
Looks good

**Pull request merged**